companies/haspa.json

```
@@ -0,0 +1,18 @@
+{
+    "slug": "haspa",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Hamburger Sparkasse AG",
+    "address": "Ecke Adolphsplatz / Gr. Burstah\n20457 Hamburg",
+    "email": "haspa@haspa.de",
+    "web": "https://haspa.de",
```

```suggestion
    "web": "https://haspa.de",
    "phone": "+49 40 35790",
    "fax": "+49 40 35793418",
```
added generic phone & fax